lms/static/js/learner_dashboard/views/course_card_view_2017.js

```diff
@@ -0,0 +1,84 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'underscore',
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'js/learner_dashboard/models/course_enroll_model',
+        'js/learner_dashboard/views/upgrade_message_view_2017',
+        'js/learner_dashboard/views/certificate_status_view_2017',
+        'js/learner_dashboard/views/course_enroll_view_2017',
+        'text!../../../templates/learner_dashboard/course_card_2017.underscore'
+    ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils,
+             EnrollModel,
+             UpgradeMessageView,
+             CertificateStatusView,
+             CourseEnrollView,
+             pageTpl
+         ) {
+             return Backbone.View.extend({
+                 className: 'program-course-card',
+
+                 tpl: HtmlUtils.template(pageTpl),
+
+                 initialize: function(options) {
+                     this.enrollModel = new EnrollModel();
+                     if (options.context) {
+                         this.urlModel = new Backbone.Model(options.context.urls);
+                         this.enrollModel.urlRoot = this.urlModel.get('commerce_api_url');
+                     }
+                     this.context = options.context || {};
+                     this.render();
+                     this.listenTo(this.model, 'change', this.render);
+                 },
+
+                 render: function() {
+                     var data = $.extend(this.model.toJSON(), {enrolled: this.context.enrolled || ''});
+                     var filledTemplate = this.tpl(data);
```

Comment:
There should only be 1 var declaration at the top of the function, comma separating the variable names.